pkg/policy/validate.go

```diff
@@ -225,12 +223,21 @@ func Validate(policy *kyverno.ClusterPolicy, client *dclient.Client, mock bool,
 					}
 				}
 			}
+
 			if rule.HasMutate() {
 				if !ruleOnlyDealsWithResourceMetaData(rule) {
 					return fmt.Errorf("policy can only deal with the metadata field of the resource if" +
 						" the rule does not match any kind")
 				}
 			}
+
+			if rule.HasVerifyImages() {
+				for _, i := range rule.VerifyImages {
+					if err := validateVerifyImagesRule(i); err != nil {
+						return err
```

Comment:
what about wrapping this err to append something like rule and image to get more verbosity?

Reply:
Good idea!